homeassistant/components/tuya/switch.py

```diff
@@ -433,6 +433,10 @@
             entity_category=ENTITY_CATEGORY_CONFIG,
         ),
     ),
+    # Smart Electricity Meter
+    # https://developer.tuya.com/en/docs/iot/smart-meter?id=Kaiuz4gv6ack7
+    "zndb": (
+        SwitchEntityDescription(key=DPCode.SWITCH, name="Switch"),),
```

Comment:
Something went wrong with the formatting here

Reply:
`black --fast tuya` changed it to this, should I revert?

Reply:
add a comma after `name="Switch",` and run black again 😉